client/my-sites/plans/jetpack-plans/product-grid/index.tsx

```diff
@@ -241,19 +258,29 @@ const ProductGrid: React.FC< ProductsGridProps > = ( {
 				</div>
 			</ProductGridSection>
 			<ProductGridSection title={ translate( 'More Products' ) }>
-				<ul className="product-grid__product-grid">
-					{ otherItems.map( getOtherItemsProductCard ) }
-
-					<li>
-						<JetpackCrmFreeCard siteId={ siteId } duration={ duration } />
-					</li>
-
-					{ showFreeCard && (
-						<li>
-							<JetpackFreeCard siteId={ siteId } urlQueryArgs={ urlQueryArgs } />
-						</li>
+				<>
+					{ showProductCategories && (
+						<div className="product-grid__category-filter">
+							<CategoryFilter
+								defaultValue={ JETPACK_SECURITY_CATEGORY }
```

Comment:
Since we're updating the url hash when the filter changes, shouldn't we then be setting the default value here to the hash (if it's present)? and if no hash is present then default to Security. If that makes sense..

In other words, in my browser, if I go directly to `/pricing#growth` then I would expect the page to load with the category filter already set to "Growth" and displaying the growth products.

Reply:
Absolutely. It was initially the case, but a change I made overwrote that behaviour. Thanks for catching that!